### PR TITLE
Fix/tests failing

### PR DESCRIPTION
**Pull request by honestbonsai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,4 @@ Whenever a file is saved, webpack rebuilds and outputs `b0x.js` in the `/dist` d
 1. Clone repo: `git clone https://github.com/b0xNetwork/protocol_contracts`
     - Note: `protocol_contracts` and `b0x.js` folders must be sibling folders
 
-1. Install dependencies for `protocol_contracts`: `npm install`
```
Comment (adrianmcli) [inline README.md]: Why was this removed? It seems like the tests still require `npm run network` to be run on the `protocol_contracts` repo.

Comment (hyperh) [reply README.md]: That is correct I will fix this line

**Comment by honestbonsai** [comment]
@adrianmcli Yea there was an attempt on that with the `network.js` file, since I thought that the network state was interfering with the tests, but they all seem to pass more reliably now. So I didn't pursue that more.

I'll remove that file since I'm not using it.

File: README.md (hunk 1 above)
Comment (honestbonsai) [reply README.md]: Will fix that
